torchvision/models/detection/transform.py

```diff
@@ -193,7 +194,7 @@ def __repr__(self):
 
 def resize_keypoints(keypoints, original_size, new_size):
     # type: (Tensor, List[int], List[int])
-    ratios = [float(s) / float(s_orig) for s, s_orig in zip(new_size, original_size)]
+    ratios = [torch.tensor(s) / torch.tensor(s_orig) for s, s_orig in zip(new_size, original_size)]
```

Comment:
can we pass the `dtype=torch.float32` here, just to make sure that we don't perform integer division?